projects/packages/waf/src/class-waf-endpoints.php

```diff
@@ -47,6 +47,34 @@ public static function register_endpoints() {
 				'permission_callback' => __CLASS__ . '::waf_permissions_callback',
 			)
 		);
+		register_rest_route(
+			'jetpack/v4',
+			'/waf/update-rules',
+			array(
+				'methods'             => WP_REST_Server::EDITABLE,
```

Comment:
Is this a POST request due to a token or something similar that authenticates? Seems to be the case here, I just notice that the endpoint itself doesn't appear to be using any posted data.